apps/api-extractor-model/src/model/ApiTypeAlias.ts

```diff
@@ -42,13 +53,37 @@ export interface IApiTypeAliasOptions extends
  *
  * @public
  */
-export class ApiTypeAlias extends ApiNameMixin(ApiReleaseTagMixin(ApiDeclaredItem)) {
+export class ApiTypeAlias extends ApiTypeParameterListMixin(ApiNameMixin(ApiReleaseTagMixin(ApiDeclaredItem))) {
+  /**
+   * An {@link Excerpt} that describes the type of the alias.
+   *
+   * @remarks
+   * In the example below, the `aliasTypeExcerpt` would correspond to the subexpression
+   * `T extends any[] ? BoxedArray<T[number]> : BoxedValue<T>;`:
```

Comment:
BTW I found it interesting that the `;` is part of the excerpt.

Reply:
I find it interesting that a number of excerpts include an empty `" "` token at the end.